### PR TITLE
Redirect users to oc-explain for references.

### PR DESCRIPTION
**Pull request by kaovilai**

### CODE REVIEW

File: docs/API_ref.md
Comment (shubham-pampattiwar) [inline docs/API_ref.md:3]: Instead of giving the instructions to install the CRDs manually, Should we just add a note that the user should make sure they have OADP operator installed on their cluster or something ?

Comment (kaovilai) [reply docs/API_ref.md:3]: I didn't mean to instruct them to install anything manually.. was just listing them out :D

Comment (kaovilai) [reply docs/API_ref.md:3]:
```suggestion
Pre-requisites: Install OADP to your cluster. before proceeding to the next steps.
```

File: docs/API_ref.md
Comment (kaovilai) [inline docs/API_ref.md:36]:
```suggestion

```

File: docs/API_ref.md
Comment (kaovilai) [inline docs/API_ref.md:5]:
```suggestion
Run `oc api-resources | grep -e 'oadp\|velero'` to get available APIs

You can use `oc explain <full-name|kind|short-name>.<fields>` to explore available APIs
```

**Comment by weshayutin** [comment]
@kaovilai for stable branches 1.1, 1.2 imho we should build on your original idea.
https://github.com/weshayutin/oadp-operator/blob/wes_api_doc/docs/API_ref.md

File: docs/API_ref.md
Comment (eemcmullan) [inline docs/API_ref.md:3]: What about listing the CRDs that this will install? So they know what they should be looking for. 
"Once the operator is installed, the following CRDs will be available:...."

Comment (kaovilai) [reply docs/API_ref.md:3]: @shubham-pampattiwar where was I
> ... giving the instructions to install the CRDs manually

([comment](https://github.com/openshift/oadp-operator/pull/990#pullrequestreview-1398549854))
in https://github.com/openshift/oadp-operator/commit/42a72811bd48048e582c64af90c1fa964d2b45fa ?

**Comment by kaovilai** [comment]
/hold

Need to discuss if we wanna output everything into .md for stable.

**Comment by kaovilai** [comment]
Consider using https://github.com/kubernetes/kubectl/issues/1419 when its implemented.

**Comment by kaovilai** [comment]
For master I think this is good, but may want to add a script to makefile here for use in generating in other branches.

**Comment by weshayutin** [comment]
@kaovilai will https://pkg.go.dev/github.com/openshift/oadp-operator@master render after this is merged?
doesn't look like it to me..
guess we need to point at a release not a branch
https://pkg.go.dev/github.com/openshift/oadp-operator?tab=versions

**Comment by kaovilai** [comment]
@weshayutin  https://pkg.go.dev/github.com/openshift/oadp-operator@master is now rendering.

After awhile the cache is deleted, and you have to request the page.

**Comment by kaovilai** [comment]
Although other branch names (oadp-1.0, etc.) do not work for pkgsite. but we can work around that by generating a tagged version.
https://github.com/golang/go/issues/46914

**Review by weshayutin — APPROVED** [review]

**Pull request merged**
